lib/src/config/v2/tui/mod.rs

```diff
@@ -0,0 +1,188 @@
+use std::path::Path;
+
+use anyhow::{Context, Result};
+use serde::{Deserialize, Serialize};
+
+use super::server::ComSettings;
+
+pub mod config_extra;
+pub mod keys;
+pub mod theme;
+
+#[derive(Debug, Clone, Deserialize, Serialize, PartialEq, Default)]
+#[serde(default)] // allow missing fields and fill them with the `..Self::default()` in this struct
+#[allow(clippy::module_name_repetitions)]
+pub struct TuiSettings {
+    pub com: MaybeComSettings,
```

Comment:
I think "com" is a bit confusing without seeing the server settings in the context of tui. Maybe we could use something like "bind" for server config and "bind_server" in tui?

Reply:
i intended to keep it the same name as the server config, being short for `communication`, as in the future i intend to maybe add another way than grpc, like a unix socket.

Reply:
> i intended to keep it the same name as the server config

Probably it is better to keep them identical. But the short form took me off guard when looking through examples, so it might be worth it to expand it to the full form. Bind should be applicable for all kinds of client/server semantics too.